modules/gdnative/gdnative/string.cpp

```diff
@@ -125,6 +122,35 @@ void GDAPI godot_string_new_with_wide_chars_and_len(godot_string *r_dest, const
 	}
 }
 
+const char GDAPI *godot_string_to_latin1_chars(const godot_string *p_self) {
+	String *self = (String *)p_self;
+	return self->ascii(true).get_data();
+}
+
+const char GDAPI *godot_string_to_utf8_chars(const godot_string *p_self) {
+	String *self = (String *)p_self;
+	return self->utf8().get_data();
+}
+
+const char16_t GDAPI *godot_string_to_utf16_chars(const godot_string *p_self) {
+	String *self = (String *)p_self;
+	return self->utf16().get_data();
+}
+
+const char32_t GDAPI *godot_string_to_utf32_chars(const godot_string *p_self) {
+	String *self = (String *)p_self;
+	return self->get_data();
+}
+
+const wchar_t GDAPI *godot_string_to_wide_chars(const godot_string *p_self) {
```

Comment:
Given `wchar_t` is not portable, are you sure we want to provide it ?
It seems simple enough for the user implement it himself, and not providing it prevent the user from using it by default

Reply:
Maybe. I'm adding for convenience and completeness. The only issue is that on Windows `wchar_t` is 2 bytes, but it is 4 bytes everywhere else.

Reply:
> on Windows `wchar_t` is 2 bytes, but it is 4 bytes everywhere else.

Windows is like that for literally everything :D